src/stock_str.rs

```diff
@@ -275,6 +275,42 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Download maximum available until %1$s"))]
     DownloadAvailability = 100,
+
+    #[strum(props(fallback = "Inbox"))]
+    Inbox = 103,
+
+    #[strum(props(fallback = "Outbox"))]
+    Outbox = 104,
```

Comment:
I would keep "Outgoing messages", because I think most users know what "Outgoing messages" are but not what an "outbox" is.

And these strings are more visible than the configuration screen which is hidden behind an "Advanced" button (although I still think that it would be nice to change them away from "Inbox" and "Outbox" in the configuration screen, too. Not so high priority though because it clearly says "IMAP" and "SMTP" directly underneath, and people who manually enter things there will know what these are)